sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -411,4 +421,101 @@ public CosmosAsyncDatabase getDatabase(String id) {
     public void close() {
         asyncDocumentClient.close();
     }
+
+    TracerProvider getTracerProvider(){
```

Comment:
Do we need this?
We can directly use `this.tracerProvider` wherever needed. Given that the `CosmosAsynClient(Builder) `constructor is the only entry point to access these API's?